setup.cfg

```diff
@@ -14,7 +14,9 @@ extend-ignore =
 	# TokenError: unterminated string literal
 	E902,
 	# Mutable default arg of type List
-	M511
+	M511,
+    # Logging and IO shadowing python's builtins
+    A005
```

Comment:
Out of curiosity, did the Lint GitHub workflow crash without it? I made this change locally because it was, but I didn't push it since it was working on github.

Reply:
Did you run using Python 3.9?

Reply:
Yes. I run on python 3.9. It is okay to have this here I believe.